### DISCUSSION TITLE
Add other uncover source in passive mode

### DISCUSSION THREAD

**Discussion by Techbrunch** [projectdiscovery/naabu#448] [Ideas]
It would be nice if naabu could also use Censys in passive mode.

**Comment by ehsandeep** [comment]
@Techbrunch shodan have [dedicated unauthenticated API ](https://internetdb.shodan.io) that accepts IPs with higher rate limit to share open ports information, but I don't think it's case with any other search engines that can be implemented.

**Comment by Techbrunch** [comment]
@ehsandeep Why was it converted to a discussion ? This is a feature request. I'm ok with the passive scan using authenticated sources.

**Reply by ehsandeep** [reply]
@Techbrunch I don't think it can be implemented without proper API support, which I don't see in any other search engine except shodan. Let me know if you have more details about the API you wanted to be supported.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
